src/sourceMachine.ts

```diff
@@ -610,7 +610,9 @@ export const makeSourceMachine = (params: {
           };
         }),
         updateURLWithMachineID: (ctx) => {
-          params.routerReplace(`/${ctx.sourceID}`);
+          params.router.replace(`/${ctx.sourceID}`, undefined, {
+            shallow: true,
```

Comment:
using shallow like this won't update meta tags in the head - it seems though like fixing this could be moved to an issue

Reply:
```suggestion
            // using shallow like this won't update meta tags in the head
            shallow: true,
```

Reply:
With yesterday changes I've solved updating meta tags in the head